python/tests/test_logistic_regression.py

```diff
@@ -998,8 +1008,23 @@ def test_multiclass(
         gpu_number=gpu_number,
         reg_param=0.1,
         tolerance=tolerance,
+        set_float32_inputs=True,
```

Comment:
set_float32_inputs=(data_type == np.float32) or similar would be simpler logic than repeating the whole instantiation.

Reply:
Need to test the case converting float64 to float32. 
Revised to use randomized value.